src/Options.jl

```diff
@@ -318,6 +319,22 @@ function Options(;
     binary_operators = map(binopmap, binary_operators)
     unary_operators = map(unaopmap, unary_operators)
 
+    diff_binary_operators = Any[]
+    diff_unary_operators = Any[]
+
+    for op in binary_operators
+        diff_op(x, y) = gradient(op, x, y)
```

Comment:
Okay added it. Now it displays the following message:

```julia
julia> options = Options(;
           binary_operators=(+, *, -, /, ^),
           unary_operators=(cos, exp, sin, myrelu),
       )
┌ Warning: Automatic differentiation has been turned off, since operator myrelu does not have well-defined gradients.
└ @ SymbolicRegression.var"../Options.jl" ~/Documents/SymbolicRegression.jl/src/Options.jl:362
```

where we have defined `myrelu` as `myrelu(x) = x > 0 ? x : 0` - which returns `nothing` from `Zygote.gradient` for negative input..